packages/apollo-voyager-conflicts/src/handleConflict.ts

```diff
@@ -0,0 +1,49 @@
+import { SyncServerError } from './detectConflict'
+
+function RETURN_TO_CLIENT (conflict: SyncServerError, currentRecord: ConflictResolutionData, client: ConflictResolutionData): SyncServerError {
+  console.warn(`Conflict detected. Server: ${currentRecord} client: ${client}`)
+  return conflict
+}
+
+function CLIENT_WINS (conflict: SyncServerError, currentRecord: ConflictResolutionData, client: ConflictResolutionData): ConflictResolutionHandler {
+  console.warn(`Conflict detected. Server: ${currentRecord} client: ${client}`)
+  const newVersion = currentRecord.version + 1
```

Comment:
This is very bad as we mixing here default strategy with default conflictResolution. 
Practically making it impossible to use anything else that default